.github/workflows/kvrocks.yaml

```diff
@@ -310,7 +313,16 @@ jobs:
         if: ${{ startsWith(matrix.image, 'centos') }}
         run: |
           yum install -y centos-release-scl-rh
-          yum install -y devtoolset-11 python3 autoconf automake wget git
+          yum install -y devtoolset-11 python3 autoconf automake wget git gcc gcc-c++
+          echo "NPROC=$(nproc)" >> $GITHUB_ENV
+          mv /usr/bin/gcc /usr/bin/gcc-4.8.5
+          ln -s /opt/rh/devtoolset-11/root/bin/gcc /usr/bin/gcc
+          mv /usr/bin/g++ /usr/bin/g++-4.8.5
+          ln -s /opt/rh/devtoolset-11/root/bin/g++ /usr/bin/g++
```

Comment:
Why we cannot run `source /opt/rh/devtoolset-11/enable` exactly here?

Reply:
I tried this, but it failed. `source /opt/rh/devtoolset-11/enable` takes effect only on current session. Maybe we can add this into `.bashrc`, i'll try it.

Reply:
> I tried this, but it failed. `source /opt/rh/devtoolset-11/enable` takes effect only on current session. Maybe we can add this into `.bashrc`, i'll try it.

I tried this by adding `echo "source /opt/rh/devtoolset-11/enable" >> /root/.bashrc` into setup stage. It didn't take effect.  :(
![image](https://github.com/apache/kvrocks/assets/85615957/0b60daea-c51f-47fa-bba6-0c7851c0c7df)

Reply:
We can add the archllinux support first, and refactoring for improvements later.

Reply:
> We can add the archllinux support first, and refactoring for improvements later.

You mean go back to edition 00db8cf?

Reply:
Or the current one should be fine. As long as CI passed and functionality correct.